src/uint256.h

```diff
@@ -91,6 +91,15 @@ class base_blob
                ((uint64_t)ptr[7]) << 56;
     }
 
+    uint32_t GetUint32(int pos) const
+    {
+        const uint8_t* ptr = data + pos * 8;
```

Comment:
why *8? that doesn't look in line with 32 bit values.

Also, please add `LE` to make clear that this is always little-endian.

Reply:
Maybe I am seeing outdated code, but it still says `* 8` here and `uint64_t` below.